source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py

```diff
@@ -235,3 +251,6 @@ def get_places(self, obj):
                 abort(404, message='Unknown places type {}'.format(type_))
             res.append(map_serializer[type_](feature).data)
         return res
+
+    def get_feed_publishers(self, _):
```

Comment:
I would be a bit more straightforward to use a Literal field here instead of a methodField (methodField is the weak way :grin: )

Reply:
@antoine-de 
something like this?
```
feed_publisher_bano = {
    "id": "bano",
    "name": "Base d'Adresses Nationale Ouverte",
    "license": "ODbL",
    "url": "http://bano.openstreetmap.fr/data/lisezmoi-bano.txt"
}

feed_publisher_osm = {
    "id": "osm",
    "license": "ODbL",
    "name": "openstreetmap",
    "url": "https://www.openstreetmap.org/copyright"
}

feed_publishers = {"feed_publihsers": [feed_publisher_bano, feed_publisher_osm]}

...

feed_publishers = LiteralField(json.dumps(feed_publishers))
```

Reply:
yes, but you don't need the json dumps

https://github.com/CanalTP/navitia/blob/dev/source/jormungandr/jormungandr/interfaces/v1/JSONSchema.py#L106-L110

Reply:
Thx!